### PR TITLE
[Excel] (Custom Functions) CDN mention

### PR DESCRIPTION
**Pull request by mscharlock**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b78dcc6](https://github.com/OfficeDev/office-js-docs-pr/pull/931/commits/b78dcc6b59ef9156743d58fc87664e7d611f01cc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-architecture.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cdn/docs/excel/custom-functions-architecture.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cdn/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904290147088749-931%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2FoOcGkdf9eeS5fvmaZljGy1%2FKeOStsazL2O3aZnePzM%3D&st=2019-04-29T01%3A43%3A15Z&se=2019-05-30T01%3A48%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-overview.md [hunk 1]
```diff
@@ -61,6 +61,12 @@ function add(first, second){
  CustomFunctions.associate("ADD", add);
 ```
 
+Note that the functions.html file, which governs the loading of the custom functions runtime, should contain the current CDN for custom functions. If you have prepared your project using the most current version of yo office, this CDN will already be present, but if you are retrofitting a previous custom function from March 2019 or earlier, you may need to copy in the code below to the functions.html page.
```
Comment (AlexJerabek) [inline docs/excel/custom-functions-overview.md]: Bold the file name here too.

Comment (AlexJerabek) [inline docs/excel/custom-functions-overview.md]: "should contain the current CDN" -> "must link to the current CDN"

Comment (AlexJerabek) [inline docs/excel/custom-functions-overview.md]: Consider breaking the last sentence up. Something like:
"Projects prepared with the current version of yo office reference the CDN, If you are retrofitting a previous custom function project from March 2019 or earlier, you need to copy in the code below to the **functions.html** page."

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0dd2e1f](https://github.com/OfficeDev/office-js-docs-pr/pull/931/commits/0dd2e1fb67972ae99b2c2a606e86dcca269b7fb8)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-6524ad99)

<a id="6524ad99"></a>
###
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/OfficeDev/office-js-docs-pr, working directory: W:\x4az-s, target branch: master, target branch commit: d37503f74d89384b8c1368d1f2f443ed7edd8e26, source commit: 0dd2e1fb67972ae99b2c2a606e86dcca269b7fb8.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904291822379946-931%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=iJGYXG6IKJdY0aResegJCeI%2Ff%2F54YIkEL9AvGwL8%2F3Q%3D&st=2019-04-29T18%3A17%3A50Z&se=2019-05-30T18%3A22%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-overview.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ function add(first, second){
  CustomFunctions.associate("ADD", add);
 ```
 
-Note that the functions.html file, which governs the loading of the custom functions runtime, should contain the current CDN for custom functions. If you have prepared your project using the most current version of yo office, this CDN will already be present, but if you are retrofitting a previous custom function from March 2019 or earlier, you may need to copy in the code below to the functions.html page.
+Note that the **functions.html** file, which governs the loading of the custom functions runtime, must link to the current CDN for custom functions. Projects prepared with the current version of the Yo Office generator reference the correct CDN. However, if you are retrofitting a previous custom function project from March 2019 or earlier, you need to copy in the code below to the **functions.html** page.
```
Comment (AlexJerabek) [inline docs/excel/custom-functions-overview.md]: Nit: remove "However"

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1ecf63f](https://github.com/OfficeDev/office-js-docs-pr/pull/931/commits/1ecf63fef0665aff174c5619a82095be38949f0f)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-c6bc3a87)

<a id="c6bc3a87"></a>
###
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/OfficeDev/office-js-docs-pr, working directory: W:\ji0t-s, target branch: master, target branch commit: d858ecead620c035fcf275b4d21e2782e4f91e18, source commit: 1ecf63fef0665aff174c5619a82095be38949f0f.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904291852254672-931%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Q%2BpPv%2FzzuScSpk3jxwa8EoWMzRqYETyZh%2FP30VDHbkw%3D&st=2019-04-29T18%3A47%3A37Z&se=2019-05-30T18%3A52%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[76a4327](https://github.com/OfficeDev/office-js-docs-pr/pull/931/commits/76a4327e39c501ee8d853a48f58096c61e1b1370)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-architecture.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cdn/docs/excel/custom-functions-architecture.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cdn/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904291852400126-931%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=8L1Bw330uIljU%2FtQvH2YF%2Bqri4uLB9lLy1ZKGecTZJ0%3D&st=2019-04-29T18%3A48%3A55Z&se=2019-05-30T18%3A53%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**